doc/api/stream.md

```diff
@@ -2376,8 +2377,12 @@ changes:
     underlying stream will _not_ be aborted if the signal is aborted. The
     callback will get called with an `AbortError`. All registered
     listeners added by this function will also be removed.
+  * `autoCleanup` {boolean} remove all registered stream listeners.
```

Comment:
I'd prefer to simplify this to just `cleanup`

Reply:
ok thanks i update the code accordingly